devlog/2024-11-20-clean-cleanl-clean.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Clean, clean, clean!
+slug: clean-clean-clean
+authors: alfi
+tags: [housekeeping, cleanup, devlog, solus]
+hide_table_of_contents: false
+---
+
+Hello everyone! Alfi here, member of Solus Cleanup Crew. It's been more than a year since I started contributing to Solus and I want to talk about it. About what I did, what we did, what we accomplished so far, and what we want to do next.
+
+<!-- truncate -->
+
+## A bit of background
+I started contributing on October 2023, when Solus participate in [Hacktoberfest](https://getsol.us/2023/10/01/solus-and-hacktoberfest-2023/). Hacktoberefest is a month-long event that encourage people to contribute to open source project in exchange for some digital merchandise. Finally, an excuse to contribute to something I use everyday. Although, as an accountant, all of this is new experience to me. It was confusing at first, but I persisted. Finally, I get the hang of it and I managed to submit around 100 pull request during the month of the event. I was very excited and motivated. During the month, a few people joined the event and contributed to Solus, but then November came and most of the contribution slowed down. I was the only one left that continue doing the beginner tasks, mostly adding homepage to packages.
+
+At the start January 2024, Jakub Nugen ([androidnisse](https://github.com/androidnisse)) comes in and start contributing to homepage task too. Worried that we would stepped on each other works, we started a private chat to organize the tasks between ourselves. Until May, the two of us have been chipping away the homepage task which at this point leaving 650 packages without homepage. Then we get more contributors joining. I decided to make a group chat to accomodate all the people joining. It was called "Solus Homepage Team" at the time. More contributors and staff members join the room, then it was renamed as "Solus Cleanup Crew" and moved under official Solus matrix space.
```

Comment:
```suggestion
At the start of January 2024, Jakub Nugen ([androidnisse](https://github.com/androidnisse)) comes in and starts contributing to the homepage task too. Worried that we would step on each other's feet, we started a private chat to organize the tasks between ourselves. Until May, the two of us had been chipping away at the homepage task which at this point left 650 packages without a homepage. Then we had more contributors joining. I decided to make a group chat to accommodate all these new people. It was called "Solus Homepage Team" at the time. More contributors and staff members joined the room, then it was renamed as "Solus Cleanup Crew" and moved under the official Solus matrix space.
```

Seems like "the homepage task" is mentioned without first explaining what it is. Maybe add a short sentence to that effect somewhere in the beginning.
(also some native speaker better double-check some of my suggestions ^^)
((David already mentioned the accommodate thing, but I thought since I was already at it I'd mark it again for completeness' sake)

Reply:
I think it was already implied by the end of the first paragraph:
"I was the only one left that continued doing the beginner tasks, mostly adding homepage to packages."
Which from then on I refer as "homepage task"